pkg/cmd/task/describe.go

```diff
@@ -42,45 +43,42 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .Task.Name }}
 
 {{decorate "inputresources" ""}}{{decorate "underline bold" "Input Resources\n"}}
 
-{{- if not .Task.Spec.Inputs }}
+{{- if not .Task.Spec.Resources }}
```

Comment:
So we won't display `Inputs.Resources` on a v1alpha1 cluster with 0.10.x right ?
I think we should… but this is gonna make the template ever more complex :sweat_smile:

Reply:
Yes, it makes sense to me.

Reply:
A lot of the logic could be taken out of the templates in functions to keep it simpler.

Reply:
Fixed to work with 0.10.x